addons/web/static/src/views/kanban/kanban_column_progressbar.scss

```diff
@@ -44,21 +32,9 @@
     .o_kanban_counter {
         transition: opacity 0.3s ease 0s;
 
-        > .o_kanban_counter_progress {
+        > .o_column_progress {
```

Comment:
this scss has not been moved to a standard column_progress.scss file since it concerns only the kanban view